spec/project/object/target_dependency_spec.rb

```diff
@@ -40,6 +40,32 @@ module ProjectSpecs
       lambda { @target_dependency.sort_recursively }.should.not.raise
     end
 
+    it 'tree hash does not contain target' do
```

Comment:
nit: could this specify that it includes the `targetProxy`?

Reply:
I thought about it, but then I'd have to include the `display_name ` and `isa` too, which makes the test name too verbose IMO :)